src/components/Error/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+
+interface ErrorProps {
+  navigateBack: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  variant?: 'Error' | '404';
+}
+
+const useStyles = makeStyles(({ spacing }) => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+  },
+  errorType: {
+    display: 'flex',
+  },
+  boldText: {
+    fontWeight: 'bold',
+    marginRight: spacing(1),
+  },
+  button: {
+    height: spacing(4),
+    width: spacing(16),
+    marginTop: spacing(2),
+  },
+}));
+
+export const Error: FC<ErrorProps> = ({ navigateBack, variant = 'Error' }) => {
+  const classes = useStyles();
+
+  const errorMessage =
+    variant === 'Error' ? (
+      <Typography variant="h3" className={classes.boldText}>
+        ERROR
+      </Typography>
+    ) : (
+      <div className={classes.errorType}>
+        <Typography variant="h3" className={classes.boldText}>
+          404
+        </Typography>
```

Comment:
I know this is from the designs but it rubs me the wrong way. There's no reason to artificially create this error code. The not found message is more than enough.

Reply:
agree, removed